Yoast/wordpress-seo
# Upgrade webpack to the latest version

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Upgraded Webpack to version 4.X
* Imported React as an external.
* Extracted the webpack runtime to a seperate file.

## Test instructions

This PR can be tested by following these steps:

* Clone this branch of the plugin by running:`git clone --single-branch -b 10914-upgrade-webpack git@github.com:Yoast/wordpress-seo.git` in a bash terminal.
* Ensure that [yarn](https://yarnpkg.com/en/docs/install#mac-stable) and [composer](https://getcomposer.org/) are installed.
* Install the dependancies by running `yarn` in the folder where the plugin was cloned in.
* Build the plugin by running `grunt artifact`. This will generate a zip file in the root folder of the plugin to install on your wordpress site.
* [Install the generated zip file on your wordpress site](https://www.wpbeginner.com/beginners-guide/step-by-step-guide-to-install-a-wordpress-plugin-for-beginners/)
* Visit the following pages and verify if there are no console errors nor misplaced items:
- [ ] Post edit page
- [ ] The configuration pages
- [ ] The Google search console page
- [ ] The Gutenberg structured data blocks
- [ ] Configuration wizard

## Quality assurance


Fixes #10914
